tests/_support/AcceptanceTester.php

```diff
@@ -156,38 +174,40 @@ public function checkAllResults()
 	/**
 	 * Selects an option in a Chosen Selector based on its id
 	 *
-	 * @param   string  $selectId  The id of the <select> element
-	 * @param   string  $option    The text in the <option> to be selected in the chosen selector
+	 * @param   string $selectId The id of the <select> element
+	 * @param   string $option   The text in the <option> to be selected in the chosen selector
 	 *
 	 * @return void
 	 */
 	public function selectOptionInChosenById($selectId, $option)
 	{
 		$chosenSelectID = $selectId . '_chzn';
-		$I = $this;
+		$I              = $this;
 		$this->comment("I open the $chosenSelectID chosen selector");
 		$I->click(['xpath' => "//div[@id='$chosenSelectID']/a/div/b"]);
 		$this->comment("I select $option");
 		$I->click(['xpath' => "//div[@id='$chosenSelectID']//li[text()='$option']"]);
 		$I->wait(1); // Gives time to chosen to close
 	}
+
 	/**
 	 * Function to Verify the Tabs on a Joomla! screen
 	 *
-	 * @param  Array  $expectedTabs   Expected Tabs on the Page
-	 * @param  Mixed  $tabsLocator    Locator for the Tabs in Edit View
+	 * @param  Array $expectedTabs Expected Tabs on the Page
+	 * @param  Mixed $tabsLocator  Locator for the Tabs in Edit View
 	 *
 	 * @return void
 	 */
 	public function verifyAvailableTabs($expectedTabs, $tabsLocator = ['xpath' => "//ul[@id='myTabTabs']/li/a"])
 	{
-		$I = $this;
+		$I                 = $this;
 		$actualArrayOfTabs = $I->grabMultiple($tabsLocator);
```

Comment:
Thanks. I will remove space and add new line after it. Lot of space is because of auto code style correction.